bin/roave-infection-static-analysis-plugin

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\InfectionStaticAnalysis\Application;
+
+use Infection\Console\Application;
+use Infection\Container;
+use PackageVersions\Versions;
+use Psalm\Config;
+use Psalm\Internal\IncludeCollector;
+use Psalm\Report;
+use Roave\InfectionStaticAnalysis\Bootstrapper;
+use Roave\InfectionStaticAnalysis\Psalm\RunStaticAnalysisAgainstMutant;
+use function define;
+use function defined;
+use function getcwd;
+
+(static function (): void {
+    $cwd = getcwd();
+
+    (static function () use ($cwd): void {
+        require_once $cwd . '/vendor/autoload.php';
+    })();
+
+    if (! defined('PSALM_VERSION')) {
+        define('PSALM_VERSION', Versions::getVersion('vimeo/psalm'));
+    }
+
+    if (! defined('PHP_PARSER_VERSION')) {
+        define('PHP_PARSER_VERSION', Versions::getVersion('nikic/php-parser'));
+    }
+
+    $psalmConfig = Config::getConfigForPath($cwd, $cwd, Report::TYPE_CONSOLE);
+
+    $psalmConfig->setIncludeCollector(new IncludeCollector());
+
+    (new Application(Bootstrapper::bootstrap(
+        Container::create(),
+        new RunStaticAnalysisAgainstMutant($psalmConfig)
```

Comment:
As this binary is psalm-specific, I would consider renaming this to `bin/roave-infection-psalm-plugin` to allow for a `bin/roave-infection-phpstan-plugin` alongside this one.